firmware/uimage.ksy

```diff
@@ -7,7 +7,7 @@ doc: |
   The new uImage format allows more flexibility in handling images of various
   types (kernel, ramdisk, etc.), it also enhances integrity protection of images
   with sha1 and md5 checksums.
-doc-ref: https://github.com/EmcraftSystems/u-boot/blob/master/include/image.h
+doc-ref: https://source.denx.de/u-boot/u-boot/-/raw/master/include/image.h
```

Comment:
Same thing as in https://github.com/kaitai-io/kaitai_struct_formats/pull/427#discussion_r588261626: please refer to permanent URLs.

```suggestion
doc-ref: https://source.denx.de/u-boot/u-boot/-/raw/e4dba4b/include/image.h
```

Reply:
oops. updated.